pages/newuser.vue

```diff
@@ -0,0 +1,63 @@
+<template lang="pug">
+  .centered-container.flex.justify-center.items-center.my-10
+    .form
+      .form-container(class="p-8 bg-white rounded-lg shadow-lg max-w-4xl w-full min-w-[900px]")
+        
+        // Header section
+        .form-header.text-center.bg-customBlue.p-6.rounded-t-lg.text-gray-100
+          h2.text-4xl.font-medium.uppercase.tracking-wider Sign Up
+          .heading-line.w-32.h-1.bg-green-400.my-2.mx-auto.rounded-sm
+        
+        // Form fields
+        .form-input.grid.gap-6.p-8.bg-white.rounded-b-lg
+          
+          // First Name Field
+          .form-element.flex.flex-col
+            label(for="first-name" class="text-lg font-semibold text-gray-800 mb-2") First Name
+            input#first-name(type="text" placeholder="First Name" required class="p-3 text-base border border-gray-300 rounded-sm transition-all duration-300 ease-in-out focus:border-blue-500 focus:ring-2 focus:ring-customBlue-500")
+          
+          // Last Name Field
+          .form-element.flex.flex-col
+            label(for="last-name" class="text-lg font-semibold text-gray-800 mb-2") Last Name
+            input#last-name(type="text" placeholder="Last Name" required class="p-3 text-base border border-gray-300 rounded-sm transition-all duration-300 ease-in-out focus:border-blue-500 focus:ring-2 focus:ring-customBlue-500")
+          
+          // Username Field
+          .form-element.flex.flex-col
+            label(for="username" class="text-lg font-semibold text-gray-800 mb-2") User Name
+            input#username(type="text" placeholder="User Name" required class="p-3 text-base border border-gray-300 rounded-sm transition-all duration-300 ease-in-out focus:border-blue-500 focus:ring-2 focus:ring-customBlue-500")
+          
+          // Password Field
+          .form-element.flex.flex-col
+            label(for="password" class="text-lg font-semibold text-gray-800 mb-2") Password
+            input#password(type="password" placeholder="Password" required class="p-3 text-base border border-gray-300 rounded-sm transition-all duration-300 ease-in-out focus:border-blue-500 focus:ring-2 focus:ring-customBlue-500")
+          
+          .div(class="flex flex-col space-y-6") 
+            label(class="block text-xl font-semibold") Sign me up as a ...
```

Comment:
This should just be role, remember this is the Admin (Rae) assigning the new users (Parents and Teachers), roles